components/list/list-item-drag-mixin.js

```diff
@@ -0,0 +1,504 @@
+import { css, html } from 'lit-element/lit-element.js';
```

Comment:
This file was reviewed in detail already in https://github.com/BrightspaceUI/core/pull/665

A good read over would be good though. Nothing bad about double checking.